specifications/api/swagger.yaml

```diff
@@ -541,6 +541,17 @@ definitions:
         <<: *walletPassphrase
         description: A master passphrase to lock and protect the wallet for sensitive operation (e.g. sending funds).
 
+  PostTransactionData: &PostTransactionData
```

Comment:
Moved from L569 and renamed "parametersPostTransaction" > "PostTransactionData"

I noticed similar naming for swagger endpoint arguments in https://github.com/input-output-hk/cardano-wallet/blob/5c7d690e5d994c931e5b437c5bdbaf222f6dc675/src/Cardano/Wallet/Api.hs#L88 so I followed the same naming scheme